server/controllers/messages.coffee

```diff
@@ -350,23 +358,43 @@ module.exports.send = (req, res, next) ->
     async.series steps, (err) ->
         return next err if err
         return next new Error('Server error') unless jdbMessage
-        res.send 200, jdbMessage.toClientObject()
+        out = jdbMessage.toClientObject()
+        out.isDraft = isDraft
+        res.send 200, out
```

Comment:
Use the new Express API:

```coffeescript
res.status(200).send out
```

By the way, if it's a 200 code, you don't need to mention it.